src/Mod/Path/PathScripts/PathDrilling.py

```diff
@@ -78,9 +78,8 @@ def initCircularHoleOperation(self, obj):
         obj.addProperty("App::PropertyBool", "DwellEnabled", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "Enable dwell"))
         obj.addProperty("App::PropertyBool", "AddTipLength", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "Calculate the tip length and subtract from final depth"))
         obj.addProperty("App::PropertyEnumeration", "ReturnLevel", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "Controls how tool retracts Default=G98"))
-        obj.ReturnLevel = ['G98', 'G99']  # this is the direction that the Contour runs
-
-        obj.addProperty("App::PropertyDistance", "RetractHeight", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "The height where feed starts and height during retract tool when path is finished"))
+        obj.ReturnLevel = ['G98', 'G99']  # Canned Cycle Return Level
+        obj.addProperty("App::PropertyDistance", "PeckRetractHeight", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "The height where feed starts and height during retract tool when path is finished while in a peck operation"))
```

Comment:
Not sure I get the requirement to rename the property. I would probably just leave it as is - also means it won't break existing ops.

Reply:
I wanted this to be more self documenting, but don't care either way so: reverted